server/raft.go

```diff
@@ -2069,23 +2079,28 @@ func (n *raft) runAsCandidate() {
 		case <-n.quit:
 			return
 		case <-elect.C:
+			n.switchToCandidate()
+			return
+		case <-tikChan:
+			// disable timeout and receipt of more votes
+			voteChan = nil
+			tikChan = nil
```

Comment:
tikChan is time.After() and will fire only once, not sure setting it to nil here does anything, but doesn't hurt.

Reply:
remove